README.md

```diff
@@ -25,46 +25,62 @@
 [![codecov](https://codecov.io/gh/apache/openwhisk/branch/master/graph/badge.svg)](https://codecov.io/gh/apache/openwhisk)
 [![Twitter](https://img.shields.io/twitter/follow/openwhisk.svg?style=social&logo=twitter)](https://twitter.com/intent/follow?screen_name=openwhisk)
 
-OpenWhisk is a cloud-first distributed event-based programming service. It provides a programming model to upload event handlers to a cloud service, and register the handlers to respond to various events. Learn more at [http://openwhisk.apache.org](http://openwhisk.apache.org).
+OpenWhisk is a serverless functions platform for building cloud applications.
+OpenWhisk offers a rich programming model for creating serverless APIs from functions,
+composing functions into serverless workflows, and connecting events to functions using rules and triggers.
+Learn more at [http://openwhisk.apache.org](http://openwhisk.apache.org).
+
+* [Quick Start](#quick-start) (Deploy and Use OpenWhisk on your machine)
+* [Deploy to Kubernetes](#deploy-to-kubernetes) (For development and production)
+* For project contributors and Docker deployments:
+  * [Deploy to Docker for Mac](./tools/macos/README.md)
+  * [Deploy to Docker for Ubuntu](./tools/ubuntu-setup/README.md)
+* [Learn Concepts and Commands](#learn-concepts-and-commands)
+* [OpenWhisk Community and Support](#openwhisk-community-and-support)
+* [Project Repository Structure](#project-repository-structure)
 
+### Quick Start
 
-* [Quick Start](#quick-start) (Docker-Compose)
-* [Native development](#native-development) (Mac and Ubuntu)
-* [Kubernetes](#kubernetes-setup)
-* [Learn concepts and commands](#learn-concepts-and-commands)
-* [Issues](#issues)
-* [Slack](#slack)
+The easiest way to start using OpenWhisk is to install the "Standalone Controller". This is a full-featured OpenWhisk stack running as a Java
+process for convenience. Serverless functions run within Docker containers.
+Please refer to [Docker's installation instructions](https://docs.docker.com/install/) to configure your system to use Docker.
 
-### Quick Start
-The easiest way to start using OpenWhisk is to get Docker installed on Mac, Windows or Linux. The [Docker website](https://docs.docker.com/install/) has detailed instructions on getting the tools installed. This does not give you a production deployment but gives you enough of the pieces to start writing functions and seeing them run.
+```
+git clone https://github.com/apache/openwhisk.git
```

Comment:
Currently I think build may fail if node is not installed. Should we highlight that node and java are needed. Also not sure if build works fine on Windows

Best would have been to have users download a prebuilt jar but for that I think we need to wait for the core repo release.

Reply:
I will add Node and Java are also required. Agree on the jar, we need to push toward the release.